src/app/components/error/error.layout.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from 'react';
+import { Box, Flex, Text, color, Button } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import GenericError from '@assets/images/generic-error.png';
+
+interface ErrorProps {
+  body: string;
+  helpText: ReactNode[];
+  onClose(): void;
+  title: string;
+}
+export function ErrorLayout(props: ErrorProps) {
+  const { body, helpText, onClose, title } = props;
+
+  return (
+    <Flex alignItems="center" flexDirection="column" px={['loose', 'unset']} width="100%">
+      <Box mt="loose">
+        <img src={GenericError} width="106px" />
+      </Box>
+      <Title fontSize={4} mt="loose">
+        {title}
+      </Title>
+      <Text
+        color={color('text-caption')}
+        fontSize="16px"
+        lineHeight="1.6"
+        mt="base"
+        textAlign="center"
+      >
+        {body}
+      </Text>
+      <Box
+        as="ul"
+        border="2px solid #EFEFF2"
+        borderRadius="12px"
+        color={color('text-caption')}
+        fontSize="14px"
+        lineHeight="1.6"
+        mt="extra-loose"
+        pb="loose"
+        pl="40px"
+        pr="loose"
+        pt="tight"
+        width="100%"
+      >
+        {helpText}
+        <Box as="li" mt="base">
+          Still stuck? Reach out to{' '}
+          <Text
+            as="button"
+            color={color('accent')}
+            // TODO: Where should this go?
+            onClick={() => {}}
```

Comment:
Yeah, I wonder. I don't think we really want a `mailto:` link, do we? 🤔

Reply:
@landitus what do we want the support email link to do?

Reply:
Yes please!